src/Infrastructure.EntityFramework/Repositories/DatabaseContext.cs

```diff
@@ -180,6 +181,8 @@ protected override void OnModelCreating(ModelBuilder builder)
         eOrganizationDomain.ToTable(nameof(OrganizationDomain));
         aWebAuthnCredential.ToTable(nameof(WebAuthnCredential));
 
+        builder.Entity<ProjectSecret>(b => b.ToTable(nameof(ProjectSecret)));
```

Comment:
Thoughts on moving this into `src/Infrastructure.EntityFramework/SecretsManager/Configurations` into its own `IEntityTypeConfiguration` file?

For this case, it isn't much configuration, but I find it easier for managing the team's specific configurations.